pkg/plugins/manager/signature/manifest.go

```diff
@@ -99,32 +98,49 @@ func readPluginManifest(body []byte) (*pluginManifest, error) {
 	return &manifest, nil
 }
 
-func Calculate(mlog log.Logger, plugin *plugins.Plugin) (plugins.Signature, error) {
-	if plugin.IsCorePlugin() {
+func Calculate(mlog log.Logger, class plugins.Class, plugin plugins.FoundPlugin) (plugins.Signature, error) {
+	if class == plugins.Core {
```

Comment:
If we decide to do this comparison instead of using the functions, I would consider switching to `class == plugins.X` vs `plugin.IsXPlugin()` in every place and remove the `IsXPlugin` functions from `pkg/plugins/plugins.go`.

I find it confusing having two ways of checking the plugin's class :smiley:

Reply:
Yeah I see what you mean. This is a bit different though I guess as we're not checking it against the plugin, but the class param directly. Will make a note of it!